Assets/MRTK/packagetemplate.json

```diff
@@ -32,6 +32,9 @@
         "SDK*",
         "Services*",
         "package.json.meta",
+        "LICENSE*",
```

Comment:
Do we need these in all the other packages too? I think I just see Foundation and Examples

Reply:
At least LICENSE and maybe CHANGELOG?

Reply:
The other packages are governed by .npmignore files. They will get picked up automatically